device-types/Cisco/N5K-C5548UP.yaml

```diff
@@ -0,0 +1,81 @@
+manufacturer: Cisco
```

Comment:
This definition was submitted in PR #23. When you create new branches, be sure to base each of them off of the `master` branch to avoid redundant changes.

Reply:
sorry about that, will check better in future.